meson.build

```diff
@@ -0,0 +1,177 @@
+project('waltham', 'c',
+        version: '0.0.0',
```

Comment:
In autotools we were already version 0.1.0.

Reply:
There were a few other locations where it was 0.0.0 instead, such as the soname; I considered that more important for backwards compatibility but I can revert if you prefer so.

Reply:
I thought soname was the only other place. We have made a 0.1.0 release, so package version should not go down. The library version was 0.0.0 and can stay that way until we start caring about it.